roles/metrics-export.lua

```diff
@@ -190,6 +190,25 @@ local function validate_http_node(node)
         check_server_httpd_role(httpd_role.DEFAULT_SERVER_NAME)
     end
 
+    if node.ssl_key_file ~= nil and type(node.ssl_key_file) ~= 'string' then
+        error("ssl_key_file sould be a string, got " .. type(node.ssl_key_file), 3)
+    end
+    if node.ssl_cert_file ~= nil and type(node.ssl_cert_file) ~= 'string' then
+        error("ssl_cert_file sould be a string, got " .. type(node.ssl_cert_file), 3)
+    end
+    if node.ssl_ca_file ~= nil and type(node.ssl_ca_file) ~= 'string' then
+        error("ssl_ca_file sould be a string, got " .. type(node.ssl_ca_file), 3)
+    end
+    if node.ssl_ciphers ~= nil and type(node.ssl_ciphers) ~= 'string' then
+        error("ssl_ciphers_file sould be a string, got " .. type(node.ssl_ciphers), 3)
+    end
+    if node.ssl_password ~= nil and type(node.ssl_password) ~= 'string' then
+        error("ssl_password sould be a string, got " .. type(node.ssl_password), 3)
+    end
+    if node.ssl_password_file ~= nil and type(node.ssl_password_file) ~= 'string' then
+        error("ssl_password_file sould be a string, got " .. type(node.ssl_password_file), 3)
+    end
```

Comment:
We need to throw an error is a `server` + a TLS option is set.

Reply:
Reworked.